pkg/cli/cli.go

```diff
@@ -70,6 +70,8 @@ type CLI struct { //nolint:maligned
 	pluginKeys []string
 	// Project version to scaffold.
 	projectVersion config.Version
+	// projectName name of project
+	projectName string
```

Comment:
Why we would need to pass the projectName here? 
The projectName is found in the PROJECT file.

Reply:
That variable gets populated by the project file [here](https://github.com/rumstead/kubebuilder/blob/e7ee3941b34e3ab56362dfaa6f945bfb15a75f8b/pkg/cli/scaffold.go#L56). 

Can I access the project name without exposing a new variable here?